### PR TITLE
Bump retext, add support for python 3.8 and 3.9 for retext plus deps

### PR DESCRIPTION
**Pull request by holgersson32644**

### CODE REVIEW

**Comment by a17r** [comment]
Tests pass?

**Comment by juippis** [comment]
Ping.

**Comment by holgersson32644** [comment]
Updated and bumped, all tests went through, waiting for a reply of mgorny wrt to a dep (asked him a few minutes ago).

File: app-text/pytextile/pytextile-4.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    	test_textile.py
+	)
+	for test in ${my_drop_tests[@]};
+		do rm "tests/$test" || die
+	done
+}
+
+python_test() {
+	pytest || die "Testsuite failed under ${EPYTHON}"
+}
```
Comment (juippis) [inline app-text/pytextile/pytextile-4.0.1.ebuild:56]: Could probably be replaced with `distutils_enable_tests`.

**Pull request closed without merging**